controllers/dspipeline_params_test.go

```diff
@@ -258,3 +260,26 @@ func TestExtractParams_CABundle(t *testing.T) {
 func strPtr(v string) *string {
 	return &v
 }
+
+func TestExtractParams_WithCustomKfpLauncherConfigMap(t *testing.T) {
+	ctx, params, client := CreateNewTestObjects()
+	cmDataExpected := map[string]string{
+		"this-is-the-only-thing": "that-should-be-in-kfp-launcher-now",
+	}
+	cm := v1.ConfigMap{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      "my-custom-kfp-launcher",
+			Namespace: "testnamespace",
+		},
+		Data: cmDataExpected,
+	}
+	err := client.Create(ctx, &cm)
+	assert.Nil(t, err)
+
+	dspa := testutil.CreateDSPAWithCustomKfpLauncherConfigMap("my-custom-kfp-launcher")
+	err = params.ExtractParams(ctx, dspa, client.Client, client.Log)
+	assert.Nil(t, err)
+
+	cmDataExpectedJson, err := json.Marshal(cmDataExpected)
+	assert.Equal(t, string(cmDataExpectedJson), params.CustomKfpLauncherConfigMapData)
```

Comment:
Same as above. (Technically it doesn't make any difference here since there's no following code.

Reply:
fixed